test/system/010-images.bats

```diff
@@ -330,4 +330,31 @@ Deleted: $pauseID"
     run_podman rm my-container --force -t 0
 }
 
+@test "podman pull image with additional store" {
+    local imstore=$PODMAN_TMPDIR/imagestore
+    local sconf=$PODMAN_TMPDIR/storage.conf
+    cat >$sconf <<EOF
+[storage]
+driver="overlay"
+
+[storage.options]
+additionalimagestores = [ "$imstore" ]
+EOF
+
+    run_podman save $IMAGE -o $PODMAN_TMPDIR/save
+    run_podman --root $imstore load -i $PODMAN_TMPDIR/save
```

Comment:
Well, my bad. I thought this might work but it doesn't. With save/load, this test passes even on `main`, indicating that this is not a test. With pull, the test passes on your PR but fails on main.

@mtrmac can you think of a skopeo incantation to copy from one local image store to another? I can't figure it out.

Reply:
Got it. This fails on main (with a very very ugly failure to clean up) and succeeds with your PR applied:
```
...
[storage.options]
additionalimagestores = [ "$imstore/root" ]
EOF

    skopeo copy containers-storage:$IMAGE \
           containers-storage:\[overlay@$imstore/root+$imstore/runroot\]$IMAGE

    CONTAINERS_STORAGE_CONF=$sconf run_podman images -a -n --format "{{.Repository}}:{{.Tag}} {{.ReadOnly}}"
    is "${lines[0]}" "$IMAGE false" "image from readonly store"
    is "${lines[1]}" "$IMAGE true" "image from readwrite store"

    CONTAINERS_STORAGE_CONF=$sconf run_podman images -a -n --format "{{.Id}}"
    id=${lines[0]}

    CONTAINERS_STORAGE_CONF=$sconf run_podman pull -q $IMAGE
    is "$output" "$id" "Should only print one line"

    run_podman --root $imstore/root rmi --all
```